solr/CHANGES.txt

```diff
@@ -8,6 +8,13 @@ https://github.com/apache/lucene-solr/blob/master/solr/solr-ref-guide/src/solr-u
 
 Consult the LUCENE_CHANGES.txt file for additional, low level, changes in this release.
 
+Improvements
+---------------------
+
+* SOLR-14014: Introducing a system property that allows users to disabled the Admin UI, which is enabled by default.
```

Comment:
[+1] Thanks for including CHANGES text!  David made a point recently about ensuring this gets included in PRs, but I still personally always forget.

[-0] Typo here: "disabled" -> "disable"

[0] This blurb should probably mention the name of the system property, so users can add it easily if they want to.